components/src/dynamo/thompson_router/management.py

```
@@ -0,0 +1,248 @@
+        app.router.add_post("/state", self._load_state)
+        app.router.add_post("/state/reset", self._reset_state)
+        app.router.add_get("/config", self._get_config)
+        app.router.add_post("/config", self._set_config)
+        app.router.add_get("/decisions", self._get_decisions)
+        app.router.add_get("/decisions/summary", self._get_decisions_summary)
+        app.router.add_get("/metrics", self._get_metrics)
+        app.router.add_post("/metrics/reset", self._reset_metrics)
+
+        self._runner = web.AppRunner(app)
+        await self._runner.setup()
+        site = web.TCPSite(self._runner, "0.0.0.0", self._port)
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)